openapi.json

```diff
@@ -1797,7 +1871,7 @@
     "/credentials/basic": {
       "get": {
         "summary": "HTTP Basic authentication",
-        "description": "This request checks whether the credentials provided in the HTTP Basic header are valid. Returns the user ID and a bearer token for authorization in subsequent API calls. It is RECOMMENDED to implement this authentication method for non-public services only.",
+        "description": "This request checks whether valid credentials are provided through HTTP Basic Authentication (HTTP header `Authorization`, with type `Basic`). Returns an access token to be used as Bearer token for authorization in subsequent API calls.\n\nIt is RECOMMENDED to implement this authentication method for non-public services only.\n\nAfter authentication, Clients SHOULD request further details about the user from `GET /me` so that they can make requests to the `/files/{user_id}` endpoints.",
```

Comment:
Now that we have the `basic//` prefix, I would clarify the bit about Bearer token a bit, to avoid the confusion whether this endpoint should return the access token with or without this prefix. E.g. something like

> Returns an access token (without `basic//` prefix) to be used in the Bearer token for authorization in subsequent API calls.

Reply:
I've added a paragraph to each authentication method about the Bearer Token format, see commit https://github.com/Open-EO/openeo-api/pull/225/commits/371511b6b1dda6d2ddc837382063855a6da5bd19 - Is this clear enough?